azure-webapp-maven-plugin/src/test/java/com/microsoft/azure/maven/webapp/utils/TestUtils.java

```diff
@@ -9,26 +9,78 @@
 import java.io.IOException;
 
 public class TestUtils {
+    private static final String clientId = System.getenv("CLIENT_ID");
+    private static final String tenantId = System.getenv("TENANT_ID");
+    private static final String key = System.getenv("KEY");
+
+    private static final boolean isWindows = System.getProperty("os.name").contains("Windows");
+
+    private static void azureLogin() throws IOException, InterruptedException {
+        final String azLoginTemplate = "az login --service-principal -u %s -p %s --tenant %s";
+        Runtime.getRuntime().exec(formatCommand(azLoginTemplate, new String[]{clientId, key, tenantId})).waitFor();
+    }
+
+    private static void azureLogout() throws IOException, InterruptedException {
+        Runtime.getRuntime().exec(formatCommand("%s", new String[]{"az logout"})).waitFor();
+    }
+
     public static void deleteAzureResourceGroup(String resourceGroupName, boolean waitForOperationFinish)
             throws InterruptedException, IOException {
-        final String clientId = System.getenv("CLIENT_ID");
-        final String tenantId = System.getenv("TENANT_ID");
-        final String key = System.getenv("KEY");
-        final String azLoginTemplate = "az login --service-principal -u %s -p %s --tenant %s";
-        final String azDelteTemplate = "az group delete -y -n %s%s";
 
-        final String[] commands = { String.format(azLoginTemplate, clientId, key, tenantId),
-                String.format(azDelteTemplate, resourceGroupName, waitForOperationFinish ? "" : " --no-wait"),
-                "az logout" };
+        final String azResourceGroupDeleteTemplate = "az group delete -y -n %s%s";
+        azureLogin();
+        Runtime.getRuntime().exec(formatCommand(azResourceGroupDeleteTemplate,
+                new String[]{resourceGroupName, waitForOperationFinish ? "" : " --no-wait"})).waitFor();
+        azureLogout();
+    }
+
+    public static void createWindowsAzureWebApp(final String resourceGroupName, final String location,
+                                                final String servicePlanName, final String webAppName)
+            throws IOException, InterruptedException {
+        final String azResourceGroupCreateTemplate = "az group create -n %s --location %s";
```

Comment:
`azResourceGroupCreateTemplate` looks like this string can be extracted as a const